docs/assembly.rst

```diff
@@ -391,12 +394,35 @@ will have a wrong impression about the stack height at label ``two``:
 .. code::
 
     {
+        let x := 8
```

Comment:
This is `x`'s binding occurrence.  In which range of the program is `x` bound?  In the whole block, or in the block after this `let`?